app/Http/Requests/DeleteUserRequest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class DeleteUserRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            "confirm_deletion" => ["required", "boolean", "accepted"],
```

Comment:
It is for what? Confirmation should be added on frontend - modal or something like that